src/Content/Conversation/Entity/Channel.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @copyright Copyright (C) 2010-2023, the Friendica project
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Content\Conversation\Entity;
+
+class Channel extends Timeline
+{
+	const WHATSHOT         = 'whatshot';
+	const FORYOU           = 'foryou';
+	const FOLLOWERS        = 'followers';
+	const SHARERSOFSHARERS = 'sharersofsharers';
+	const IMAGE            = 'image';
+	const VIDEO            = 'video';
+	const AUDIO            = 'audio';
+	const LANGUAGE         = 'language';
+
+	public function __construct(string $code = null, string $label = null, string $description = null, string $accessKey = null, string $path = null, int $uid = null, string $includeTags = null, string $excludeTags = null, string $fullTextSearch = null, int $mediaType = null, int $circle = null)
```

Comment:
If the constructor is identical to the parent class, you don't need to duplicate it.

Reply:
Done